mobile-widgets/qmlinterface.h

```diff
@@ -217,7 +217,7 @@ class QMLInterface : public QObject {
 	bool verbatim_plan() { return prefs.verbatim_plan; }
 	bool display_variations() { return prefs.display_variations; }
 
-	const QStringList &diveSummaryText() { return diveSummary::diveSummaryText; }
+	const QStringList diveSummaryText();
```

Comment:
As far as I can see, the code is just move into the cpp file (of course due to the static variable), but it breaks with how the rest is implemented, without (it seems to me) no real reason.